File: sdk/js/src/enums/ChatProviderTypeEnum.js
/**
 * Enum for chat provider types.
 *
 * @readonly
 * @enum {string}
 */
export const ChatProviderTypeEnum = Object.freeze({
  /** OpenAI provider. */
  OpenAI: 'OpenAI',

  /** Ollama provider. */
  Ollama: 'Ollama',

  /** Google Gemini provider. */
  Gemini: 'Gemini',

  /** Anthropic provider. */
  Anthropic: 'Anthropic',

  /** Voyage AI provider (embeddings only). */
  VoyageAI: 'VoyageAI',
});

File: sdk/python/src/litegraph_sdk/enums/chat_provider_type_enum.py
from enum import Enum


class ChatProviderType_Enum(str, Enum):
    """
    Chat provider type enum.
    """
    OpenAI = "OpenAI"
    Ollama = "Ollama"
    Gemini = "Gemini"
    Anthropic = "Anthropic"
    VoyageAI = "VoyageAI"
